WordPress/Sniffs/VIP/SessionVariableUsageSniff.php

```diff
@@ -46,6 +69,24 @@ public function register() {
 	 * @return void
 	 */
 	public function process_token( $stackPtr ) {
+		if ( false === $this->thrown['DeprecatedSniff'] ) {
+			$this->thrown['DeprecatedSniff'] = $this->phpcsFile->addWarning(
+				'The "WordPress.VIP.SessionVariableUsageSniff" sniff has been deprecated. Please update your custom ruleset.',
+				0,
+				'DeprecatedSniff'
+			);
+		}
+
+		if ( ! empty( $this->exclude )
```

Comment:
This sniff does not have any public properties which can be changed, so this complete code block for the `FoundPropertyForDeprecatedSniff` warning can be removed.